### PR TITLE
Modified index and script to use fetch() to request content from server

### PR DESCRIPTION
**Pull request by thuavu**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -19,7 +19,8 @@
         </div>
 
         <center>
-            <a href="/data"><button class="buttonBox"><h3>Comments</h3></button></a>
+            <!--<a href="/data"><button class="buttonBox"><h3>Comments</h3></button></a>-->
```
Comment (TsueiCat) [inline portfolio/src/main/webapp/index.html]: Remove commented-out line.

**Review by TsueiCat — CHANGES_REQUESTED** [review]
Nice work! Could you also include a screenshot of this change (how your portfolio looks when you click on the button) to the description?

**Comment by thuavu** [comment]
image
image

@TsueiCat 
The button essentially request go to the /data page which fetch content from the server and add it to a HTML page.

**Review by TsueiCat — CHANGES_REQUESTED** [review]
Please also resolve comments made! :)

**Comment by thuavu** [comment]
I think I got them now

**Review by TsueiCat — COMMENTED** [review]
Please add the commit change to this PR!

**Comment by thuavu** [comment]
@TsueiCat I sent the commit accidentally to the other branch.

**Review by TsueiCat — COMMENTED** [review]
Just add another commit to this branch that contains the change :) When you merge this branch to master, before merging the other pull request you'll want to run 'git pull origin master' to get the latest copy and the commit you made there for this PR will have no diff in code changes (for removing the commented-out line).

**Comment by thuavu** [comment]
@TsueiCat I'm currently getting a merge conflict. How do I fix that?

**Review by TsueiCat — APPROVED** [review]
Good job sorting out problems yourself! 😸

**Comment by thuavu** [comment]
@TsueiCat Haha, it was mainly just trying different things until they work.

**Pull request merged**